add issue template for new books

Chocorean opened the pull request:
This PR adds a simple template for helping players to gather the relevant information. It comes with a link to a short video showing how to get the book ID.
Note the link should eventually be replaced to another one pointing to Vazkii's repo, if kept at all. (I don't believe it is possible to directly add a video in the template)

Let me know what you think about it, if there is anything you wish to change please let me know. Preview: https://github.com/Chocorean/AkashicTome/issues

Chocorean committed:
add issue template for new books

Uraneptus commented:
Thanks looks really good. What should the user put in the mod version field?

Uraneptus commented:
Hmm yeah you're probably right, knowing the mod version for which the user looked up the mod id would be helpful! Could you maybe extend the description of that field a bit more? So it's clear that we need the version the requester used when looking for the id. Oh and maybe add a placeholder for that field as well showing a quick example.

Another thing, could you add a placeholder in the title field where the requester can add the mod name? Just so I have a better overview when looking for a specific issue.

Chocorean committed:
Update book_request.yml

Update title to include mod's name, and update the mod's version field (description + placeholder)

Chocorean committed:
Update book_request.yml

Add missing quote

Chocorean committed:
Update book_request.yml

Make all fields mandatory

Uraneptus commented:
Thank you that's perfect! Oh and yeah I'm the only maintainer atm, so assigning me is fine.

About the video, could I upload it to my yt as not-listed and then exchange the link? I can credit you in the video description.

Chocorean committed:
Update book_request.yml

Update mod version placeholder

Chocorean commented:
I updated the placeholder, hoping this time it is what you expect, sorry for the misunderstanding.
Regarding the video, this is fine with me!

Chocorean committed:
Update book_request.yml

Update placeholder so they all belong to roots classic and not a mix of roots classic and roots 4

Uraneptus committed:
Update book_request.yml
